app/modules/DateTimeModule.scala

```diff
@@ -20,15 +20,19 @@ import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 
 import com.google.inject.AbstractModule
-import modules.DateTimeFormatterModule.NotificationsPageFormatter
+import javax.inject.{Inject, Provider, Singleton}
 
-class DateTimeFormatterModule extends AbstractModule {
-  override def configure(): Unit =
-    bind(classOf[DateTimeFormatter]).toInstance(NotificationsPageFormatter)
-}
+class DateTimeModule extends AbstractModule {
+  def timezone = ZoneId.of("Europe/London")
```

Comment:
Why it's `def` not `val`?

Reply:
for override - this is called only one on injector configuration.